source/connect/favorite-connections/import-export-ui/import.txt

```diff
@@ -0,0 +1,56 @@
+.. _compass-import-connections-ui:
+
+==============================================
+Import Favorite Connections on |compass-short|
+==============================================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can import favorite connections on |compass|. This lets you use saved 
+connections from other workspaces or connections used by other team members.
+
+Prerequisite
+~~~~~~~~~~~~
+
+Before you import favorite connections, you must :ref:`export the
+connections <compass-export-connections-ui>`.
+
+Procedure
+~~~~~~~~~
+
+To import favorite connections from a JSON file into |compass-short|:
+
+.. procedure::
+    :style: normal
+    
+    .. step:: Hover over :guilabel:`Saved Connections` in the left-side 
+        navigation and click the :guilabel:`Ellipses (...)` button.
+
+    .. step:: Select :guilabel:`Import saved connections` to open the import 
+        dialog box.
+
+    .. step:: Under :guilabel:`Source File`, click :guilabel:`Select a file...` 
+        to select the exported JSON file you want to import.
+
+    .. step:: If your file is encrypted with a password, enter the passphrase 
+        under :guilabel:`Decryption Password`.
+    
+    .. step:: Select the connection names you want to 
+        import. 
+        
+        If the connection is already saved, |compass-short| overwrites the 
+        existing connection with the imported connection.
+
+    .. step:: Click 
+        :guilabel:`Import`
+        
+Results
+~~~~~~~
+
+Once the connections are imported, you will see them on the left-side navigation 
```

Comment:
[nit suggestion]

Consider replacing "Once" with "After". I think "After" does a better job conveying "first this thing happens, then this next thing happens."